tests/utils/service_control_utils.go

```diff
@@ -663,7 +664,10 @@ func VerifyReport(body []byte, er *ExpectedReport) error {
 	}
 
 	if er.ApiKeyInOperationAndLogEntry != "" && er.ApiKeyInLogEntryOnly != "" {
-		return fmt.Errorf("cannot set both `ApiKeyInOperationAndLogEntry` and `ApiKeyInLogEntryOnly`")
+		return fmt.Errorf("cannot set both `ApiKeyInOperationAndLogEntry` and `ApiKeyInLogEntryOnly`, they imply different api key validity")
+	}
+	if er.ApiKeyInOperationAndLogEntry != "" && er.JwtAuth != "" {
```

Comment:
We can support both ApiKey and JwtAuth.  Why do we want to reject such case?

Reply:
Renamed to make it clearer. This is for setting `CredentialId` in operation. If API Key is valid, then it will be the credential id, so can't specify both.